scripts/sync-rbac.sh

```diff
@@ -8,7 +8,7 @@
 # role etc, requires manual syncing
 # requires yq (Go), not python-yq
 
-set -e -o pipefail
+set -e -o pipefail extglob
```

Comment:
I'm curious what `extglob` is and why we need it.

Reply:
In the end the shell option wasn't needed. Removed in 1454ed69f3c41f4b59f1a28f63503313de276bf7

From `man bash`:
```
      If the extglob shell option is enabled using the shopt builtin, several extended pattern
       matching operators are recognized.  In the following description, a pattern-list is a list of
       one or more patterns separated by a |.  Composite patterns may be formed using one or more of
       the following sub-patterns:

              ?(pattern-list)
                     Matches zero or one occurrence of the given patterns
              *(pattern-list)
                     Matches zero or more occurrences of the given patterns
              +(pattern-list)
                     Matches one or more occurrences of the given patterns
              @(pattern-list)
                     Matches one of the given patterns
```